convert2rhel/actions/report.py

```diff
@@ -254,3 +262,33 @@ def format_report_section_heading(status_code):
 
     heading = "{highlight} {status_header} {highlight}".format(highlight=highlight, status_header=status_header)
     return heading
+
+
+def summary_as_txt(results):
+    """
+    Print the report to txt file. Used mainly by Satellite.
+    Accepts the data preformatted by summary function.
+
+    There is no special formatting needed, just the output of the checks.
+    The data are sorted from ERROR to INFO, SUCCESS aren't included.
+    """
+    txt_result = ""
+
+    combined_results_and_message = get_combined_results_and_message(results)
+
+    combined_results_and_message = find_actions_of_severity(
+        combined_results_and_message, "INFO", level_for_combined_action_data
+    )
+    combined_results_and_message = sorted(combined_results_and_message, key=lambda item: item[1]["level"], reverse=True)
+
+    for message_id, combined_result in combined_results_and_message:
+        entry = format_action_status_message(combined_result["level"], message_id[0], message_id[1], combined_result)
+        entry = colorize(entry, _STATUS_TO_COLOR[combined_result["level"]])
```

Comment:
Note: IIRC, the acceptance criteria for the text file for satellite says that it should never be colorized.

Reply:
I asked the same to Adam earlier this week and he told me that the folks asked for the ASCII codes in the text file